calico-enterprise_versioned_docs/version-3.16/releases.json

```diff
@@ -13,243 +13,495 @@
+        "version": "v3.16.1"
       },
       "l7-collector": {
         "image": "tigera/l7-collector",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "license-agent": {
         "image": "tigera/license-agent",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "tigera-cni": {
         "image": "tigera/cni",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "firewall-integration": {
         "image": "tigera/firewall-integration",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "egress-gateway": {
         "image": "tigera/egress-gateway",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "honeypod": {
         "image": "tigera/honeypod",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "honeypod-exp-service": {
         "image": "tigera/honeypod-exp-service",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "honeypod-controller": {
         "image": "tigera/honeypod-controller",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "key-cert-provisioner": {
         "image": "tigera/key-cert-provisioner",
-        "version": "v3.16.0",
+        "version": "v3.16.1",
         "registry": "quay.io"
       },
       "anomaly_detection_jobs": {
         "image": "tigera/anomaly_detection_jobs",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "anomaly-detection-api": {
         "image": "tigera/anomaly-detection-api",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "elasticsearch-metrics": {
         "image": "tigera/elasticsearch-metrics",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "packetcapture": {
         "image": "tigera/packetcapture",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "prometheus": {
         "image": "tigera/prometheus",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "coreos-prometheus": {
         "version": "v2.32.0"
       },
       "prometheus-operator": {
         "image": "tigera/prometheus-operator",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "prometheus-config-reloader": {
         "image": "tigera/prometheus-config-reloader",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "tigera-prometheus-service": {
         "image": "tigera/prometheus-service",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "es-gateway": {
         "image": "tigera/es-gateway",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "deep-packet-inspection": {
         "image": "tigera/deep-packet-inspection",
-        "version": "v3.16.0"
+        "version": "v3.16.1"
       },
       "eck-elasticsearch-operator": {
         "version": "2.5.0"
```

Comment:
Is this updated to v2.6.1 for CE v3.16.1?

Reply:
@rene-dekker can you confirm this please?

Reply:
Jiawei is correct, I pushed a commit.